packages/firebase/rollup.config.js

```diff
@@ -149,17 +155,67 @@ const componentBuilds = pkg.components
   })
   .reduce((a, b) => a.concat(b), []);
 
+const firestoreBuilds = [
+  {
+    input: `firestore/index.ts`,
+    output: [
+      {
+        file: resolve('firestore', firestorePkg.main),
+        format: 'cjs',
+        sourcemap: true
+      },
+      {
+        file: resolve('firestore', firestorePkg.module),
+        format: 'es',
+        sourcemap: true
+      }
+    ],
+    plugins,
```

Comment:
Do we not need

```
uglify({
           output: {
             ascii_only: true // escape unicode chars
            }	               
})
```

Reply:
The index.ts is just `import '@firebase/firestore';` (so is the output), I think we can skip it here.